### PR TITLE
25 feature21 remake booking

### LINKED ISSUE #25: feature#21 Remake Booking
# REQUIREMENTS
  -Make booking views
  -Use redux

# DESIGN
booking

### PR DESCRIPTION
**Pull request by FranciscoSenestrari**
# Description
Apply booking views
 - Create flights 
 - Summit flight to Firebase 
 - Refresh HomeScreen flight for user 

  ## Screenshots 
If applicable, include screenshots of the results or screenshots that help to see changes
booking
Screenshot_1657581814

### CODE REVIEW

**Review by daniel-yocupicio — DISMISSED** [review]

File: App/src/Hook/useFligths.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        setIsLoading(false);
+      } else {
+        alert('no hay mas datos');
+        setIsSuccess(true);
+        setIsLoading(false);
+      }
+    } catch (error) {
+      setIsLoading(false)
+      setIsSuccess(false)
+      console.log(error);
```
Comment (YilKanda) [inline App/src/Hook/useFligths.js:30]: Don't leave debugging code like console.log() in your GitHub code.

File: App/src/Hook/useFligths.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      setFlights(arrayFlights);
+    } else {
+      alert('no hay mas datos');
+    }
+    return usersQuerySnapshot;
+  };
+
+  return {fligths};
+}
+*/
```
Comment (YilKanda) [inline App/src/Hook/useFligths.js:53]: If you don't occupy a code, don't comment it, you need to remove it to keep your code as clean as possible on GitHub.

File: App/src/component/Header/Header.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 15,
+    height: 50,
+  },
+  button: {
+    height: 40,
+    width: 40,
+  },
+});
```
Comment (YilKanda) [inline App/src/component/Header/Header.js:32]: Call styles from an external file

File: App/src/database/data.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    
+    {
+        "key": 10,
+        "value": "Brasil",
+        "country":"Brasil",
+        "iso_airport":"BRZ"
+    }
+    
+]
```
Comment (YilKanda) [inline App/src/database/data.json:70]: Add a blank line

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by daniel-yocupicio — APPROVED** [review]

File: App/src/component/Header/Header.js (hunk 3 above)
Comment (FranciscoSenestrari) [reply App/src/component/Header/Header.js]: files fixed

**Pull request merged**